ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml

```diff
@@ -17,7 +17,7 @@
 # .\ConfigureRemotingForAnsible.ps1 -ForceNewSSLCert
 # .\ConfigureRemotingForAnsible.ps1 -SkipNetworkProfileCheck
 
-- hosts: "{{ groups['Vendor_groups'] | default('build*win*:test*win*') }}"
+- hosts: all
```

Comment:
Please undo this line.  This topic needs to be addressed in another issue which I'll open now.

Reply:
Already reverted.